packages/zosjobs/__tests__/__unit__/MonitorJobs.unit.test.ts

```diff
@@ -12,19 +12,22 @@
 jest.mock("../../src/GetJobs");
 jest.mock("@zowe/core-for-zowe-sdk");
 
-import { ImperativeError } from "@zowe/imperative";
+import { ImperativeError, Session } from "@zowe/imperative";
 import { JOB_STATUS, MonitorJobs, GetJobs } from "../../src";
 import { sleep } from "@zowe/core-for-zowe-sdk";
 import { IMonitorJobWaitForParms } from "../../src/doc/input/IMonitorJobWaitForParms";
 import { IJob } from "../../src/doc/response/IJob";
 
+// Longer timeout for async poll test
+const LONGER_TIMEOUT = 20000;
+
 describe("MonitorJobs", () => {
     // Use this so that editors don't complain about us accessing private stuff
     const privateMonitorJobs = MonitorJobs as any;
 
     /*************************************************************/
     // Unit tests for the file constants and defaults
-    describe("constant defaults", () => {
+    describe("constant defaults", () => { //GOOD
```

Comment:
Should these `//GOOD` comments be removed?

Reply:
yes lol